modules/oe_content_news/oe_content_news.post_update.php

```diff
@@ -23,3 +25,28 @@ function oe_content_news_post_update_00001_update_field_labels(array &$sandbox):
     $field_config->save();
   }
 }
+
+/**
+ * Create oe_reference field in the news content type.
+ */
+function oe_content_news_post_update_00002(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_news') . '/config/post_updates/00002_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
```

Comment:
The field doesn't get created on my local.

Reply:
How do you test it? Have you cleared cache after update?